subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -50,34 +52,28 @@ class Libero {
         finalizeConfig(config)
 
         Date date = new Date()
-        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HHmmss'Z'");
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HHmmss'Z'")
         config.projectProperties.date = date
         config.projectProperties.timestamp = dateFormat.format(date)
 
-        // Replace property references prior to running
         config.resolveProperties()
+        git.close()
+        config.projectProperties.computed = true
+    }
 
-        if (!options.force) {
-
-            println "Preparing for Release Cycle!"
-            printConfig(options, config)
-
-            //TODO: should probably move this to the cli class
-            def USER_CONFIRMED = System.console().readLine 'Do you wish to continue? [y/N]: ' as String
-            switch (USER_CONFIRMED) {
-                case "n":
-                    System.exit(1)
-                    break
-                case "y":
-                    executeRelease(options, config, git)
-                    break
-                default:
-                    println "Response must be either [y/n]"
-                    System.exit(1)
-            }
-        } else {
-            executeRelease(options, config, git)
+    /**
+     * Starts the release process
+     * @param options
+     * @param config
+     */
+    public void run(Options options, Config config) {
+        if (!config.projectProperties.computed) {
```

Comment:
Any specific reason we can't recompute every time and have to keep a `computed` flag on the `Config` object?